node/node.go

```diff
@@ -877,9 +980,26 @@ func (n *Node) runManager(ctx context.Context, securityConfig *ca.SecurityConfig
 	connCtx, connCancel := context.WithCancel(ctx)
 	defer connCancel()
 
+	// launch a goroutine that will manage our local connection to the the
+	// manager from the agent. Remember the ready channel from above, which is
```

Comment:
Nitpick:  `run` creates two ready channels, one for a manager and one for the agent, so it might be clearer to specify that this closes the manager ready channel, which was created in `run`.